doc/modules/sgd.rst

```diff
@@ -547,8 +547,8 @@ The code is written in Cython.
        <https://leon.bottou.org/projects/sgd>`_ L. Bottou - Website, 2010.
 
    .. [#2] `"Pegasos: Primal estimated sub-gradient solver for svm"
-      <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.74.8513>`_
       S. Shalev-Shwartz, Y. Singer, N. Srebro - In Proceedings of ICML '07.
+      <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.74.8513>`
```

Comment:
Please revert this one, there is no `:doi:` directive here so this is a [sphinx external link](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links)